src/go/rdctl/pkg/config/config.go

```diff
@@ -51,17 +55,29 @@ var (
 	configDir         string
 	configPath        string
 	defaultConfigPath string
+	configFile        = "rd-engine.json"
+	rdSubDir          = "rancher-desktop"
 )
 
 // DefineGlobalFlags sets up the global flags, available for all sub-commands
 func DefineGlobalFlags(rootCmd *cobra.Command) {
 	var err error
 
-	configDir, err = os.UserConfigDir()
-	if err != nil {
-		log.Fatal("Can't get config-dir: ", err)
+	switch runtime.GOOS {
```

Comment:
Rather than have two branches here, please refactor into a function and put the Windows implementation in `config_windows.go` and the Linux one in `config_linux.go`. (Since it's in the same package, there's no need to export them.)

(This would also mean it's easier to write tests.)